config/environments/development.rb

```diff
@@ -28,6 +28,13 @@
   config.action_mailer.smtp_settings = { address: ENV['SMTP_HOST'] || 'localhost', port: 1025 }
   config.action_mailer.show_previews = IdentityConfig.store.rails_mailer_previews_enabled
 
+  config.view_component.show_previews = IdentityConfig.store.component_previews_enabled
+  if IdentityConfig.store.component_previews_enabled
+    config.view_component.preview_paths = [Rails.root.join('spec', 'components', 'previews')]
+    config.view_component.default_preview_layout = 'component_preview'
+    config.lookbook.auto_refresh = false
+  end
```

Comment:
would we want to move this to `application.rb`? or should we just do that after the various JS, CSP errors are fixed and we actually could run this in a prod environment?

Reply:
Good question. I'm struggling to recall if there was any particular reason I had placed it here, since it does seem to be handling the feature flag anyways.

Reply:
Moved configuration in 7d72079.